awscli/examples/rum/put-rum-metrics-destination.rst

```diff
@@ -0,0 +1,10 @@
+**To creates or updates a destination to receive extended metrics from CloudWatch RUM**
+
+The following ``put-rum-metrics-destination`` creates or updates a destination to receive extended metrics from CloudWatch RUM. If the command succeeds, no output is returned. ::
+
+	aws rum put-rum-metrics-destination \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general